salesforce/my-conservation-life/force-app/main/default/staticresources/leafletMarkerCluster/leafletMarerClusterDefault.css

```diff
@@ -0,0 +1,60 @@
+.marker-cluster-small {
```

Comment:
This file could use some newlines between classes and the closing `}` moved one tab to the left for styling

Reply:
I strongly agree with the closing braces moving to the left